nilearn/datasets/atlas.py

```diff
@@ -618,3 +618,104 @@ def fetch_atlas_aal(version='SPM12', data_dir=None, url=None, resume=True,
               'labels': labels_dict}
 
     return Bunch(**params)
+
+
+def fetch_atlas_basc_multiscale(version='sym', data_dir=None,
```

Comment:
BASC is a method. We should probably be more specific in the name, something like `basc_multiscale_2015` to avoid name collision if another atlas is generated from another dataset using the same method. Apart from that small detail, +1 for merging.